packages/x-components/src/composables/__tests__/use-on-display.spec.ts

```diff
@@ -0,0 +1,284 @@
+import { ref, nextTick } from 'vue';
+import { useElementVisibility } from '@vueuse/core';
+import { TaggingRequest } from '@empathyco/x-types';
+import { useEmitDisplayEvent, useOnDisplay } from '../use-on-display';
+import { use$x } from '../use-$x';
+import { WireMetadata } from '../../wiring';
+
+jest.mock('@vueuse/core', () => ({
+  useElementVisibility: jest.fn()
+}));
+
+const refElementVisibility = ref(false);
+(useElementVisibility as jest.Mock).mockReturnValue(refElementVisibility);
+
+jest.mock('../use-$x', () => ({
+  use$x: jest.fn()
+}));
+
+const $xEmitSpy = jest.fn();
+(use$x as jest.Mock).mockReturnValue({
+  emit: $xEmitSpy
+});
+
+describe(`testing ${useOnDisplay.name} composable`, () => {
+  beforeEach(() => {
+    refElementVisibility.value = false;
+  });
+
+  function renderUseOnDisplayTester({
+    element = document.createElement('div'),
+    triggerOnce = true
+  }: RenderUseOnDisplayTesterOptions = {}): RenderUseOnDisplayTesterAPI {
+    const callbackSpy = jest.fn();
+
+    const { isElementVisible, unwatchDisplay } = useOnDisplay({
+      element,
+      callback: callbackSpy,
+      ...(!triggerOnce && { triggerOnce })
```

Comment:
So If I send `null` it doesn't send anything to the method, which doesn't make sense because I have `true` by default, so I'm actually not testing if the composable sets `triggerOnce = true` by default because I was doing it manually in this function.